docs/hotkeys.md

```diff
@@ -105,6 +105,6 @@
 |Cut backwards to the start of the current word|<kbd>Ctrl</kbd> + <kbd>w</kbd> / <kbd>Meta</kbd> + <kbd>Backspace</kbd>|
 |Cut the current line|<kbd>Meta</kbd> + <kbd>x</kbd>|
 |Paste last cut section|<kbd>Ctrl</kbd> + <kbd>y</kbd>|
-|Delete previous character (to left)|<kbd>Ctrl</kbd> + <kbd>h</kbd>|
-|Transpose characters|<kbd>Ctrl</kbd> + <kbd>t</kbd>|
+|Delete character behind cursor|<kbd>Ctrl</kbd> + <kbd>h</kbd>|
+|Swap with previous character|<kbd>Ctrl</kbd> + <kbd>t</kbd>|
```

Comment:
My only nit with this change is that the other changes are more consistent with using 'previous', so maybe we just drop the `(to left)`.